e2e/content-services/components/permissions-component.e2e.ts

```diff
@@ -1,413 +0,0 @@
-            await permissionsPage.addPermissionsDialog.checkAddPermissionDialogIsDisplayed();
-            await permissionsPage.addPermissionsDialog.checkSearchUserInputIsDisplayed();
-            await permissionsPage.addPermissionsDialog.searchUserOrGroup(filePermissionUser.username);
-            await permissionsPage.addPermissionsDialog.checkResultListIsDisplayed();
-            await permissionsPage.addPermissionsDialog.checkUserOrGroupIsDisplayed('EVERYONE');
-            await permissionsPage.addPermissionsDialog.searchUserOrGroup('somerandomtext');
-            await permissionsPage.addPermissionsDialog.checkResultListIsDisplayed();
-            await permissionsPage.addPermissionsDialog.checkUserOrGroupIsDisplayed('EVERYONE');
-        });
-
-        it('should be able to toggle the inherited permission', async () => {
-            await permissionsPage.checkPermissionListDisplayed();
-            await expect(await permissionsPage.isInherited()).toBe(true, 'Inherited permission should be on');
-            await permissionsPage.toggleInheritPermission();
-            await expect(await snackbarPage.getSnackBarMessage()).toContain('Disabled inherited permission', 'Disabled notification not shown');
-            await snackbarPage.waitForSnackBarToClose();
-            await expect(await permissionsPage.isInherited()).toBe(false, 'Inherited permission should be off');
-        });
-    });
-
-    describe('Changing and duplicate Permissions', () => {
-
-        beforeEach(async () => {
-            await apiService.login(fileOwnerUser.username, fileOwnerUser.password);
-            file = await uploadActions.uploadFile(fileModel.location, fileModel.name, '-my-');
-            await loginPage.login(fileOwnerUser.username, fileOwnerUser.password);
-            await contentServicesPage.goToDocumentList();
-            await contentServicesPage.checkContentIsDisplayed(fileModel.name);
-            await contentServicesPage.checkSelectedSiteIsDisplayed('My files');
-            await contentList.rightClickOnRow(fileModel.name);
-            await contentServicesPage.pressContextMenuActionNamed('Permission');
-            await permissionsPage.checkPermissionManagerDisplayed();
-            await permissionsPage.addPermissionButton.waitVisible();
-            await permissionsPage.addPermissionsDialog.clickAddPermissionButton();
-            await permissionsPage.addPermissionsDialog.checkAddPermissionDialogIsDisplayed();
-            await permissionsPage.addPermissionsDialog.checkSearchUserInputIsDisplayed();
-            await permissionsPage.addPermissionsDialog.searchUserOrGroup(filePermissionUser.firstName);
-            await permissionsPage.addPermissionsDialog.checkResultListIsDisplayed();
-            await permissionsPage.addPermissionsDialog.clickUserOrGroup(filePermissionUser.firstName);
-            await permissionsPage.addPermissionsDialog.selectRole(filePermissionUser.fullName, 'Contributor');
```

Comment:
Do we need fullName getter? After these changes I don't see usages so probably can be removed.

Reply:
removed the whole file, we already have this in the apps, and this one is testing demo-shell specific implementation